src/app/orderConfirmationDelivery/page.tsx

```diff
@@ -1,72 +1,101 @@
   AddressText,
   DateText,
+  CenterBox,
+  AddressDiv,
 } from './styles';
 
-import { Product } from '../../schema/schema';
+import { Product, User, Address } from '../../schema/schema';
 
 export default function OrderConfirmationDelivery() {
   const [Cart, setCart] = useState<Product[]>([]);
+  const [user, setUser] = useState<User>();
+  const [userAddress, setUserAddress] = useState<Address>();
 
   useEffect(() => {
     async function fetchProducts() {
-      const data = (await fetchCartItems()) as Product[];
-      setCart(data);
+      const cartItems = (await fetchCartItemsWithQuantity()) as Product[];
+      setCart(cartItems);
+    }
+
+    async function setUserDetails() {
+      const fetchedUser = await fetchUser();
+      setUser(fetchedUser);
+      const address = await fetchUserAddress(fetchedUser.id);
+      setUserAddress(address);
     }
+
     fetchProducts();
+    setUserDetails();
   }, []);
 
   return (
     <div>
       <NavBar />
-      <GlobalStyle />
+
       <BackButton destination="./storefront" />
-      <OutterBox>
-        <HeaderText>Thank you, Ethan. Your order has been placed.</HeaderText>
-        <OutterFavoriteDiv>
-          <DateText>Date: November 23.2023</DateText>
-          <ScrollDiv>
-            {Cart.map(cartItem => (
-              <FavoriteDiv key={cartItem.id}>
-                <img
-                  src={cartItem.photo}
-                  alt={cartItem.name}
-                  style={{
-                    width: '150px',
-                    height: '150px',
-                    marginLeft: '80px',
-                    marginRight: '100px',
-                  }}
-                />
-                <LabelBox>
-                  <Label>{cartItem.name}</Label>
-                  <p>Category: {cartItem.category}</p>
-                </LabelBox>
-              </FavoriteDiv>
-            ))}
-            <AddressText>
-              Shipping Address: 123 Telegraph Ave, Berkeley, 94704
-            </AddressText>
-          </ScrollDiv>
-        </OutterFavoriteDiv>
-      </OutterBox>
+      <CenterBox>
+        <OutterBox>
+          <HeaderText>
+            <Heading3Bold>
+              Thank you, {user?.first_name}. Your order has been placed.
+            </Heading3Bold>
+          </HeaderText>
+          <OutterFavoriteDiv>
+            <DateText>Order No. {user?.cart_id}</DateText>
+            <ScrollDiv>
+              {Cart.map(cartItem => (
+                <FavoriteDiv key={cartItem.id}>
+                  <img
+                    src={cartItem.photo}
+                    alt={cartItem.name}
+                    style={{
+                      width: '150px',
+                      height: '150px',
+                      marginLeft: '80px',
+                      marginRight: '100px',
+                    }}
+                  />
+                  <LabelBox>
+                    <Label>{cartItem.name}</Label>
+                    <p>Category: {cartItem.category}</p>
```

Comment:
can you change this so that its using the text components? I notice that the one above is using label but make sure that label is either coming from the global text file or using one the global text files